src/main/frontend/db/model.cljs

```diff
@@ -1417,12 +1417,13 @@
   (when-let [db (conn/get-conn repo)]
     (when-not (string/blank? page)
       (let [page (util/page-name-sanity-lc (string/trim page))
-            page-exist? (db-utils/entity [:block/name page])
-            ids (->> (d/datoms db :aevt :block/name)
-                     (filter (fn [datom]
-                               (when-not page-exist?
-                                 (string/ends-with? (:v datom) (str "/" page)))))
-                     (map :e))]
+            page-exist? (db-utils/entity repo [:block/name page])
```

Comment:
We should pass the same repo (connection would be better) to all db calls. It's easy to forget to pass the repo which is not a great default behavior for db-utils helpers. Passing different repos/connections can lead to subtle bugs